src/types.cpp

```diff
@@ -56,6 +58,32 @@ CassInet cass_inet_init_v6(const cass_uint8_t* address) {
   return inet;
 }
 
+void cass_inet_string(CassInet inet, char* output) {
+  uv_inet_ntop(inet.address_length == CASS_INET_V4_LENGTH ? AF_INET : AF_INET6,
+               inet.address,
+               output, CASS_INET_STRING_LENGTH);
+}
+
+CassError cass_inet_from_string(const char* str, CassInet* output) {
+#if UV_VERSION_MAJOR == 0
+  if (uv_inet_pton(AF_INET, str, output->address).code == UV_OK) {
```

Comment:
@avalanche123 libuv implements custom implementations of those functions. https://github.com/libuv/libuv/blob/v1.x/src/inet.c#L161-L313